src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -688,6 +688,20 @@ public override Accessibility DeclaredAccessibility
             }
         }
 
+        public bool AreLocalsZeroed
+        {
+            get
+            {
+                var data = this.GetDecodedWellKnownAttributeData();
+                if (data == null)
+                {
+                    return ContainingType.AreLocalsZeroed;
+                }
+
+                return !data.HasSkipLocalsInitAttribute && ContainingType.AreLocalsZeroed;
```

Comment:
Consider calling `ContainingType.AreLocalsZeroed` in only one code path. Perhaps:
```
var data = ...;
return data?.HasSkipLocalsInitAttribute != true && ContainingType.AreLocalsZeroed;
```
```

Reply:
Good suggestion. I'll fix that in #25673.